Directory.Build.props

```diff
@@ -70,7 +70,7 @@
     <MicrosoftDataSQLiteVersion>2.0.0</MicrosoftDataSQLiteVersion>
     <MicrosoftPowerShell5ReferenceAssembliesVersion>1.1.0</MicrosoftPowerShell5ReferenceAssembliesVersion>
     <MicrosoftServiceFabricServicesVersion>3.0.456</MicrosoftServiceFabricServicesVersion>
-    <WindowsAzureStorageVersion>8.2.1</WindowsAzureStorageVersion>
+    <WindowsAzureStorageVersion>9.1.0</WindowsAzureStorageVersion>
```

Comment:
Are these upgrades of Azure Storage and Json.Net required for the fix to work? We try not to force latest versions of dependencies on Orleans users, so that they can more easily choose which versions they are comfortable with.

Reply:
I got compile errors and had to upgrade the Azure Storage and Newtonsoft for the build to work.